src/OpenFeature.Contrib.Providers.GOFeatureFlag/OpenFeature.Contrib.Providers.GOFeatureFlag.csproj

```diff
@@ -15,8 +15,12 @@
   <PackageReference Include="System.Net.Http" Version="4.3.4" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="ZiggyCreatures.FusionCache" Version="2.2.0" />

```

Comment:
Another suggestion, please lower the required version of this dependency. The reason is, if a consumer is using 2.1, if they want to upgrade GoFF, they would need to upgrade to 2.2. My advise is to keep as lower as possible so no consumer is forced to upgrade. Other possible solution is to use `PrivateAssets` but I'm not that familiar with the full consequences.